coder_sniffer/DrupalPractice/ruleset.xml

```diff
@@ -12,10 +12,21 @@
     <exclude-pattern>*</exclude-pattern>
   </rule>
 
-  <rule ref="DrupalPractice.CodeAnalysis.VariableAnalysis.UndefinedVariable">
-    <!-- Do not run this sniff on template files. -->
+  <!-- How to include this in a better way???? -->
+  <rule ref="vendor/sirbrillig/phpcs-variable-analysis/VariableAnalysis/Sniffs/CodeAnalysis/VariableAnalysisSniff.php">
```

Comment:
This is ugly. What's the right way to do this?

Reply:
I've changed this in the latest PR to add the old DrupalPractice sniff back but only as a class that extends the vendor class. This seems to work quite well.